### ISSUE TITLE
Classification model and preview error

### ISSUE THREAD

**Issue by djaygier** [ultralytics/ultralytics#4976]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Export

### Bug

When training a classification model, i've tried each model and it keeps giving error "An error has accured. Please try again later." when using the model preview. When downloading the actual model in tflite, quantized or not, it gives from my 17 classes only 2 classes that work, and both of those are far from correct. Only class 1 and 4, with class 4 working only rarely. My dataset is around 12000 images and i train for ~40 epochs.

Im running the tflite model in flutter_tflite, in which other models trained with object detections work perfectly, the model is certainly corrupt.

Its an ant identification model, the ants are all different genus so theres a lot of difference between the species.

The val images seem to be correct and working as expected.

![download (2)](https://github.com/ultralytics/ultralytics/assets/85443953/67b044ee-af09-4ca6-a2dd-e7678cc79435)

![download (1)](https://github.com/ultralytics/ultralytics/assets/85443953/c28d2b52-ae96-43f9-9057-70b8b1d98fc5)

![download](https://github.com/ultralytics/ultralytics/assets/85443953/434a8654-bad1-47f8-bdf8-cdef59979cbe)

I'd also like to point out the loss going into negative.

<img width="1201" alt="Screenshot 2023-09-19 at 12 13 26" src="https://github.com/ultralytics/ultralytics/assets/85443953/cd279e7b-1f4a-41ff-8720-24e2439f986b">

heres the error it gives in the preview

<img width="1343" alt="Screenshot 2023-09-19 at 12 15 41" src="https://github.com/ultralytics/ultralytics/assets/85443953/b4032802-827b-4802-bc41-a5c87d0ea51f">

### Environment

- Any YOLOv8 classification model available
- ultralytics 8.0.181
- RTX 4090, own agent.
- CUDA 11.7, but most likely other cuda versions too.

### Minimal Reproducible Example

from ultralytics import YOLO, checks, hub

checks()
hub.login('') // my login here
model = YOLO('https://hub.ultralytics.com/models/') // my model here
model.train()

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@djaygier thanks for the bug report! Those losses are not negative, they are expressed in scientific notation. See https://en.wikipedia.org/wiki/Scientific_notation

For the Ultralytics HUB preview screen I'll loop in @kalenmike who is our lead developer there. Classification support is relatively new so there may be some issues as we work to implement and improve support there.
